translationtable/orphanet.yaml

```diff
@@ -0,0 +1,53 @@
+# Orphanet stores variant-disease associations
```

Comment:
maps of maps.  had not been expecting that.  
there is a function `Source.resolve()`  that will not know what to do with this.
I see it solves a scoping problem, that will be good to keep solved,  
 but I will have to think about the down stream consequences.

and precomposing the keys: 
`"Biomarker tested in gene phenotype": "is_marker_for"`
`"Disease-causing germline mutation(s) (gain of function) in gene phenotype": "causes_condition"`

is cumbersome in a different way